RELEASE.md

```diff
@@ -1,3 +1,32 @@
+## Release 0.0.15
+
+## New features 
+- Multimodal tensor combination (https://github.com/marqo-ai/marqo/pull/332, https://github.com/marqo-ai/marqo/pull/355). Combine image and text data into a single vector! Multimodal combination objects can be added as Marqo document fields. This can be used to encode text metadata into image vectors. See usage [here](https://docs.marqo.ai/0.0.15/Advanced-Usage/document_fields/#multimodal-combination-object).
```

Comment:
"This can be used to encode text metadata into image vectors. " -> "For example, this can be used to encode text metadata into image vectors."

Reply:
updated.